mimirtool.rb

```diff
@@ -0,0 +1,39 @@
+# typed: false
+# frozen_string_literal: true
+
+# This file was generated by GoReleaser. DO NOT EDIT.
+class Mimirtool < Formula
+  desc "Tools for interacting with Mimir"
+  homepage "https://grafana.com"
+  version "2.1.0"
+
+  on_macos do
+    if Hardware::CPU.arm?
```

Comment:
This should be possible with "bottles", you could see the `grafana-agent.rb` formula for reference, although what you're doing here should work for now.

Reply:
I am not that experienced with bottles, but wouldn't that mean it would need to build `mimirtool` outside of `mimir`? The `grafana-agent` is built as a standalone process so its a bit easier. 

Sadly I just pulling releases from: 
https://github.com/grafana/mimir/releases

Reply:
I finally have some spare cycles to put into this, is there anything I can do to help push this along? 👍🏽